tests/bitv/bitlist-interval001.smt2

```diff
@@ -0,0 +1,5 @@
+(set-logic ALL)
```

Comment:
I tend to default to using `(set-logic ALL)` now because I copy-paste tests to test things during development or debugging and Dolmen complains if you use `QF_BV` but later use quantifiers or integers.

(Also Why3 always uses `(set-logic ALL)` so we might as well test with this)